src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CompoundAssignmentOperator.cs

```diff
@@ -603,6 +604,11 @@ private BoundExpression TransformCompoundAssignmentLHS(BoundExpression originalL
             return variableTemp;
         }
 
+        private static bool IsInvariantArray(TypeSymbol type)
+        {
+            return (type as ArrayTypeSymbol)?.ElementType?.IsSealed == true;
```

Comment:
>? [](start = 57, length = 1)

It feels like the second conditional access is redundant.

Reply:
Updated.